content/docs/studio/user-guide/model-registry/add-a-model.md

```diff
@@ -0,0 +1,59 @@
+# Add a model
+
+You can add models from any ML project to the model registry. To add a model to
+your model registry, Iterative Studio creates an annotation for it in an
+`artifacts.yaml` file in your Git repository. If you are using the GTO command
+line tool, you can also add models from the [CLI][gto]. To add models using
+Iterative Studio, watch this tutorial video or read on below:
+
+https://youtu.be/szzv4ZXmYAs
+
+1. Click on `Add a model`
+2. Select the project to which you want to add the model. If your model file or
+   the DVC file for your model exists in a Git repository, select the project
+   that connects to this Git repository. If your model file resides in a remote
+   storage (eg, S3, GCS, etc.), select the project whose Git repository you want
+   to add the model to.
+
+   <admon>
+
+   Only repositories that you have connected to Iterative Studio are available
+   in the `Add a model` form. To connect your desired repository to Iterative
+   Studio, go to the `Projects` tab and
+   [create a project that connects to this Git repository](/doc/studio/user-guide/projects-and-experiments/create-a-project).
+   Then you can come back to the model registry and add the model.
+
+   </admon>
+
+3. Enter the path of the model file as follows:
+
+   - If the model file is in the Git repository, enter the relative path of the
+     model from the repository root.
+   - If the model file is in a remote storage but is DVC-tracked, enter the path
+     of the DVC file for your model.
+   - If the model file is in a remote storage and is not DVC-tracked, enter the
+     absolute path of the model file.
+   - If you use [MLEM] to save your model, use the path to the binary file that
+     MLEM generates. After you have run `mlem init`, Iterative Studio will be
+     able to parse the `.mlem` file to extract model metadata.
```

Comment:
Shoot, this is a problem... `mlem init` is linking to the `dvc init` ref...

Reply:
I guess it's a docs engine bug (we should fix separately). For now this should do the trick (committing):

```suggestion
     MLEM generates. After you have run
     [`mlem init`](https://mlem.ai/doc/command-reference/init), Iterative
     Studio will be able to parse the `.mlem` file to extract model metadata.
```

Reply:
> docs engine bug (we should fix separately) 

Extracted to https://github.com/iterative/dvc.org/issues/3809

> this should do the trick

Yep! It works now 👍🏼